.github/workflows/release.yml

```diff
@@ -1,260 +1,103 @@
-# Copyright 2022-2023, axodotdev
-# SPDX-License-Identifier: MIT or Apache-2.0
-#
-# CI that:
-#
-# * checks for a Git Tag that looks like a release
-# * builds artifacts with cargo-dist (archives, installers, hashes)
-# * uploads those artifacts to temporary workflow zip
-# * on success, uploads the artifacts to a Github Release
-#
-# Note that the Github Release will be created with a generated
-# title/body based on your changelogs.
+name: Build, Sign, and Release
```

Comment:
This is now builds and signs artefacts for every PR. It should be limited to release branches I guess